tests/snuba/api/endpoints/test_organization_events_spans_performance.py

```diff
@@ -836,6 +887,48 @@ def test_group_filter(self, mock_raw_snql_query):
             in mock_raw_snql_query.call_args_list[0][0][0].where
         )
 
+    def test_min_exclusive_time_filter(self):
+        self.create_event()
+
+        with self.feature(self.FEATURES):
+            response = self.client.get(
+                self.url,
+                data={
+                    "project": self.project.id,
+                    "min_exclusive_time": 4,
+                },
+            )
+
+        assert response.status_code == 200, response.content
+        assert response.data == []
```

Comment:
@udameli These test cases can be more useful to assert that an event is within the filter bounds, and as well as asserting that an event is out of bounds wrt to the filters.

Reply:
updated!